sketcherMinimizerAtom.cpp

```diff
@@ -389,7 +389,7 @@ void sketcherMinimizerAtom::writeStereoChemistry() // sets stereochemistry for
         return;
     }
     size_t n = neighbors.size();
```

Comment:
It's not valgrind, @d-b-w: the build failed (and everyting after it) because `n` is now unused.